includes/class-pw-admin-promowares-api.php

```
@@ -0,0 +1,1115 @@
+            delete_post_meta($woo_product_id, '_pw_aggregated_data_cache_time');
+            return false;
+        }
+
+        // 返回缓存的数据
+        $decoded_data = json_decode($cached_data, true);
+        if (json_last_error() === JSON_ERROR_NONE) {
+            return $decoded_data;
+        }
+
+        return false;
+    }
```

PHPMD (2.15.0)

Avoid unused private methods such as 'get_cached_product_data'. (Unused Code Rules)

(UnusedPrivateMethod)